### ISSUE TITLE
Train_imagenet setting

### ISSUE THREAD

**Issue by ashaw596** [quark0/darts#49]
Hi, I just wanted to verify the imagenet training arguments you guys used since the defaults seem a bit strange. Do you run it with python train_imagenet.py --auxiliary? 1000 epochs seems a bit large. And with the learning rate decay, the end lr is 5.91199783e-14

**Comment by quark0** [comment]
We stopped early at epoch 250. I will fix it in the next commit.

**Comment by quark0** [comment]
Fixed in f276dd346a09ae3160f8e3aca5c7b193fda1da37.
